pubmed_dl/bin/pubmed-dl

```diff
@@ -23,17 +23,7 @@ settings = Settings()
 logging.basicConfig(level=settings.loglevel)
 
 
-def main(start_date: str, end_date: str, request_file: str):
-    """
-    Main requires 3 arguments.
```

Comment:
I would still provide a very brief docstring that describes the CLI, e.g. "Downloads batches of articles from PubMed published between `start_date` and `end_date`. Write the resulting abstracts to `request_file`".

Reply:
sure, just updated with basic info